indico/modules/events/management/templates/contribution_person_list.html

```diff
@@ -28,7 +28,7 @@
             </div>
         </div>
     </div>
-    <div class="form-group">
+    <div class="form-group table-content">
```

Comment:
This way the `<thead>` is also scrolled, meaning that you cannot see the header after you scroll. In the particular case you can easily guess what each column represents, but in principal the header shouldn't scroll.

![image](https://cloud.githubusercontent.com/assets/1579899/19159862/224773b6-8bee-11e6-87e6-2b21ebb886bf.png)